src/meta/src/stream/scheduler.rs

```diff
@@ -268,7 +268,7 @@ impl Scheduler {
         fragment.vnode_mapping = Some(ParallelUnitMapping {
             original_indices,
             data,
-            ..Default::default()
+            fragment_id: fragment.fragment_id,
```

Comment:
👍🏻, please also help to remove this line: https://github.com/risingwavelabs/risingwave/blob/29758c11d6a6eb44d065fa6ab0d879ecd1870aff/src/meta/src/manager/catalog/fragment.rs#L169